### PR TITLE
Please accept me now!

### PR DESCRIPTION
**Pull request by philschulz**

### CODE REVIEW

File: chapter3/chapter3.tex [hunk 1]
```diff
@@ -106,240 +111,261 @@ \section{What is a random variable?}
...
-From the above definition we see that the random variables need to share an underlying sample space. This also gives us more
-insight into the real power of RVs: in applications we usually do not care at all about the underlying sample space but only about
-certain quantities captured by the RVs. In fact, we may not even be sure what the underlying sample space is and how we would
-draw samples from it. Therefore, we simply define a probability distribution over the random variables of interest. Since by definition
-the distribution is linked to a probability space, we can rest assured that we are working with a suitable sample space without
-having to bother what that space looks like.
+From the above definition, we see that the random variables need to
+share an underlying sample space. This also gives us more insight into
+the real power of RVs: in applications, we usually do not care at all
+about the underlying sample space but only about certain quantities
```
Comment (cschaffner) [inline chapter3/chapter3.tex]: only about the quantities

File: chapter3/chapter3.tex [hunk 2]
```diff
@@ -106,240 +111,261 @@ \section{What is a random variable?}
...
+captured by the RVs. Therefore, we define a probability distribution over the
+random variables of interest. Since by definition the distribution has to fulfil conditions that would allow us to interpret it
```
Comment (cschaffner) [inline chapter3/chapter3.tex]: "would"?
that allows us to interpret it

File: chapter3/chapter3.tex [hunk 3]
```diff
@@ -106,240 +111,261 @@ \section{What is a random variable?}
...
+as a probability measure on a probability space, we can use it without worrying
+about the underlying sample space too much. \chris{hmmm, not sure about this statement}\philip{I tried to formulate this a bit
```
Comment (cschaffner) [inline chapter3/chapter3.tex]: "too much"?
sounds vague, we can leave it out.

File: chapter3/chapter3.tex [hunk 4]
```diff
@@ -106,240 +111,261 @@ \section{What is a random variable?}
...
+By countable additivity, we simply need to sum up the probabilities of these intersections. Observe that those probabilities correspond exactly to $ P(X=x,Y=y_{1}), \ldots, P(X=x,Y=y_{1}) $. Hence, we get to the following equality:
 
 \begin{equation} \label{simpleMarginal}
-P_{X}(X=x) = \overset{n}{\underset{i=1}{\sum}} P_{XY}(X=x,Y=y_{i}) 
+P(X=x) = \overset{n}{\underset{i=1}{\sum}} P(X=x,Y=y_{i}) 
 \end{equation}
 
-Now if we set $ X = (Z_{1}, \ldots, Z_{m}) $, we can recover $ P_{Z_{1}^{m}} $ from $ P_{YZ_{1}^{m}} $ in the same way. Likewise,
-if we set $ Y = (Z_{1}, \ldots, Z_{m}) $ we get the generalization of \ref{simpleMarginal}. We will assume that each $ Z_{j} $
+If we set $ X = (Z_{1}, \ldots, Z_{m}) $, we can recover $ P_{Z_{1}^{m}} $ from $ P_{Z_{1}^{m}Y} $ in the same way.\chris{so?}\philip{Sorry, I do not understand your comment here.} Likewise,
```
Comment (cschaffner) [inline chapter3/chapter3.tex:375]: sorry for my too short comment, I somehow did not understand what this sentence adds in terms of scientific content. Is it such a common case that it needs to mention explicitly? It's totally obvious to me that you can do the same thing no matter what the RV X is.
Ah, I see that you are kind of using it as bridge to the other case where you set Y to be all these Z's...
still, I think we should skip the first part (setting X to Z_1^m)

File: chapter3/chapter3.tex [hunk 5]
```diff
@@ -429,81 +460,125 @@ \section{Two important distributions}
...
+\begin{cases}
+\theta & \mbox{ if $x=1$} \, , \\
+1-\theta & \mbox{ if $x=0$} \, .
+\end{cases}
 \end{equation}
 
-The Bernoulli is defined for one coin flip, or more generally for the drawing of a coloured ball from an urn containing balls
-of exactly two colours. Now what if we draw balls repeatedly with replacement? We will get a sequence of coloured balls. 
+The Bernoulli is defined for one coin flip, or more generally for the
+drawing of a coloured ball from an (very large) urn containing balls
```
Comment (cschaffner) [inline chapter3/chapter3.tex]: a (very large) urn

**Comment by cschaffner** [comment]
maybe the bars in the bar-plot could be a thicker? so that the thin red line does not cover them entirely?

**Comment by cschaffner** [comment]
Thanks a lot for all the work, @philschulz . It looks pretty good to me on first sight!

File: chapter3/chapter3.tex (hunk 4 above)
Comment (philschulz) [inline chapter3/chapter3.tex:375]: Yes, it is obvious, but I am assuming here that people have never worked with random variables before and do not know that it is "allowed" to a assign a random vector to a new variable. That's why I would rather leave it in.

**Comment by philschulz** [comment]
Hey Christian,

I will incorporate your comments and try to resolve the merge conflict either today or tomorrow. Making the bars in the plot bigger is no problem, I will just adjust the R script.

Have a nice weekend!

**Comment by cschaffner** [comment]
@philschulz , I have some more changes that you can find in https://github.com/BasicProbability/LectureNotes/pull/15
I guess you will have to merge them manually again...

**Comment by philschulz** [comment]
Hey Christian,

I incorporated your proposed changes and removed the merge conflicts. The pull request can now be accepted.

Here is what remains to do: 

1) I have made a couple of inline comments myself, mostly in response to yours. If you regard the problems as resolved, just erase the comments. 

2) Make the usual chapter3.tex and chapter3_forInclude.tex partition. This should only be done AFTER this request has been accepted. 

3) Compile the pdfs after this request has been accepted.

4) Update script and website.

One last remark: Merge conflicts seem to ensue whenever we update the pdfs (that's also why the pdfs in this pull request have not been updated). For the future we should probably only update the remote pdfs once we have a final version.

File: chapter3/chapter3.tex [hunk 6]
```diff
@@ -198,11 +227,11 @@ \section{Expectation and Variance}
...
 of the probabilities as weights, then the center of mass is the point at which you could put an infinitely thin spike into the
 plank from underneath such that the plank maintains perfect balance. In Figure~\ref{binomplot} this so-called expectation
-is indicated by the blue line. \chris{the spike could be drawn like in~\url{http://www.math.uah.edu/stat/expect/DiscreteCenterMass.png}}
+is indicated by an arrow that can be interpreted as visualising the spike that we can plug underneath the probability plank.
 
-\begin{Definition}
+\begin{Definition}[Expectation]
 The \textbf{expectation} of a random variable $ X $ with respect to the distribution $ P $ is defined as
-$$ \E[X] := \sum_{x \in \supp(X)} x P(X=x) $$ \, .
+$$ \E[X] := \sum_{x \in \supp(X)} x P(X=x) $$ \, . \philip{Are you sure you want to have this dot here? It seems to unnecessarily inflate the box.}
```
Comment (cschaffner) [inline chapter3/chapter3.tex:234]: the dot should have been before the $$

File: chapter3/chapter3.tex [hunk 7]
```diff
@@ -379,15 +419,14 @@ \section{Joint and conditional distributions}
...
+if $P_{XY} = P_X P_Y$, i.e.\ $\forall x \in \supp(X), \forall y \in \supp(Y): P(X=x, Y=y) = P(X=x)P(Y=y) $.
 \end{Definition}
 
 As with events, independence is equivalent to $ P_{X|Y} = P_{X} $ (and
-to $P_{Y|X}=P_Y$). Moreover, independence makes it much easier to calculate the expectation and variance of jointly distributed random variables. We have to be careful, though. We
-cannot just compute the expectation of the joint random variables
-themselves\chris{what do you mean?} as it is unclear what this would even be.
-What we can do is to compute the expectation of functions of them. The standard choice in the case of expectation
+to $P_{Y|X}=P_Y$). Moreover, independence makes it much easier to calculate the expectation and variance of jointly distributed random variables. We have to be careful, though. Jointly distributed random variables do not take on a joint value. Rather, each of the joint RVs still takes on its own value.\chris{what do you mean?}
+This is why in the joint case we have to compute the expectation of functions of the joint RVs.\philip{Is this better?} The standard choice in the case of expectation
```
Comment (cschaffner) [inline chapter3/chapter3.tex:429]: I've inserted a foonote, see master

File: chapter3/chapter3.tex [hunk 8]
```diff
@@ -492,14 +529,14 @@ \section{Two\chris{or more?} Important Distributions}
...
-  Bernoulli would be about \emph{counting} the number of failures.}
-
-The second\chris{there are already two distributions introduced above,
-  Bernoulli and Binomial} important distribution that we get to know today is the \textbf{multinomial distribution}. It is basically a generalization
+Other applications include the assessment of failure rates. Say you run a company and a customer asks you to supply $ k $ items within a week.
+The capacity of your company allows you to produce at most $ n > k $ items during that week. At the same time you know that each item that you 
+produce has a probability of being faulty. Obviously, you cannot sell faulty items to your customer. The Binomial distribution allows you to
+compute that probability that you will be able to meet the customer's demand. Based on this you can evaluate whether or not it is reasonable to
+accept the customer's order.\chris{I don't understand the example,
+  Bernoulli would be about \emph{counting} the number of failures.}\philip{Is this one better?}
```
Comment (cschaffner) [inline chapter3/chapter3.tex:537]: much better!

**Pull request merged**